src/com/projectkorra/projectkorra/earthbending/combo/EarthDomeOthers.java

```diff
@@ -31,17 +32,18 @@ public class EarthDomeOthers extends EarthAbility implements ComboAbility {
 	public EarthDomeOthers(final Player player) {
 		super(player);
 
-		if (this.bPlayer.isOnCooldown("EarthDome")) {
+		if (!this.bPlayer.canBendIgnoreBinds(this) ||
+			this.bPlayer.isOnCooldown("EarthDome")) {
```

Comment:
I believe that canBend also checks cooldowns, so you probably won't need to provide it again

Reply:
Awesome

Reply:
leaving canBend to check for the cooldown will make it check for EarthDomeOthers, not EarthDome, correct? i don't think they're the same

Reply:
That's a good point, I'll test it. It's been so long since I worked on it that I can't remember.

Reply:
nvm i didn't look at the most recent change i just assumed the parts after the || were removed

Reply:
it should work bc you used EarthDome for the canBendIgnoreBinds instead of `this`